.github/workflows/sandbox.yaml

```diff
@@ -0,0 +1,141 @@
+name: Sanbox image to build on push to sandbox/<sandbox identifier> branch
+
+on:
+  push:
+    branches:
+      - sandbox/**
+  pull_request:
+    types:
+      - opened
+
+permissions:
+  contents: write
+  id-token: write
+  pull-requests: write
+  issues: write
+
+
+env:
+  ALPINE_IMAGE: alpine:3.18.3
```

Comment:
can we update that to v3.20.2?

Reply:
not in a scope of this task as it might bring breaking changes

Reply:
@vsukhin is that safe to upgrade?

Reply:
It should be. But I would check it on dev